src/warnet/server.py

```diff
@@ -480,11 +492,13 @@ def thread_start(wn, lock: threading.Lock):
                 raise ServerError(message=message, code=CONFIG_DIR_ALREADY_EXISTS)
 
         try:
-            wn = Warnet.from_graph_file(graph_file, config_dir, network, self.backend)
-            t = threading.Thread(target=lambda: thread_start(wn, self.image_build_lock))
+            self.warnets[network] = Warnet.from_graph_file(
+                graph_file, config_dir, network, self.backend
+            )
+            t = threading.Thread(target=lambda: thread_start(self, network))
             t.daemon = True
             t.start()
-            return wn._warnet_dict_representation()
+            return self.warnets[network]._warnet_dict_representation()
```

Comment:
okay... or you could keep the local `wn` variable and add it to `self.warnets` right after `start()`

Reply:
I think it's the same thing? Just referencing the dict here